<define-tag description>bufferoverløb</define-tag>
<define-tag moreinfo>
<p>Denne sikkerhedsbulletin korrigerer DSA 458-2 der forårsagede a problem i
rutinen gethostbyaddr.</p>	

<p>Den oprindelige bulletins tekst var:</p>	
		        
<blockquote>
<p>Sebastian Schmidt har opdaget en bufferoverløbsfejl i Pythons 
getaddrinfo-funktion, hvilket kunne muliggøre at en IPv6-adresse, leveret via
DNS af en fjernangriber, kunne hukommelse i stakken.</p>

<p>Denne fejl findes kun i python 2.2 og 2.2.1, og kun når understøttelse af 
IPv6 er slået fra.  Pakken python2.2 i Debian "woody" opfylder disse 
betingelser (pakken "python" gør ikke).</p>
</blockquote>

<p>I den stabile distribution (woody), er denne fejl rettet i version 
2.2.1-4.6.</p>

<p>Distributionen testing (sarge) og den ustabile distribution (sid) er ikke 
påvirket af dette problem.</p>

<p>Vi anbefaler at du opdaterer dine python2.2-pakker.</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2004/dsa-458.data"
#use wml::debian::translation-check translation="1.6" mindelta="1"
